ToDoList-Redux/Binders/CheckmarkCellViewModel.swift

```diff
@@ -10,37 +10,17 @@ import Foundation
 import SwiftRex
 
 extension CheckmarkCellView {
-    
-
     // MARK: - STATE
     struct State: Equatable {
-        let id: String
         let title: String
         let imageName: String
 
-        static var empty = State(id: UUID().uuidString,
-                                 title: "Type to create a new task...",
+        static var empty = State(title: "Type to create a new task...",
```

Comment:
In this case I removed the mocks from the View Models because, as I'm using the real app reducer, I need to go through the full store. Benefit is that I don't have to rewrite Reducers.